openbb_sdk/sdk/core/openbb_core/app/command_runner.py

```diff
@@ -373,6 +374,13 @@ def run(
                 "timestamp": timestamp,
             }
 
+        # Based on user configuration for data output
+        # we return the data in the desired format
+        if execution_context.user_settings.preferences.data_output == "df":
+            return obbject.to_dataframe()
```

Comment:
I think doing the conversion here will break the API since it relies on `OBBject` as output and we avoid returning DataFrames there.

I'd rather do this in directly in the python UI. https://github.com/OpenBB-finance/OpenBBTerminal/tree/feature/openbb-sdk-v4/openbb_sdk/openbb/package

Reply:
![Screenshot 2023-09-11 at 11 33 01](https://github.com/OpenBB-finance/OpenBBTerminal/assets/79287829/fd73c6f0-2a49-40f7-8d20-4a59601c8ff4)